perception/util/visualizer/dynamic_object_visualization/CMakeLists.txt

```diff
@@ -1,46 +1,22 @@
-cmake_minimum_required(VERSION 3.0.2)
+cmake_minimum_required(VERSION 3.5)
 project(dynamic_object_visualization)
 
-add_compile_options(-std=c++14)
+find_package(ament_cmake_auto REQUIRED)
+ament_auto_find_build_dependencies()
 
-find_package(catkin REQUIRED COMPONENTS
-  roscpp
-  autoware_perception_msgs
-  geometry_msgs
-  sensor_msgs
-  visualization_msgs
-)
-
-catkin_package(INCLUDE_DIRS include
-  CATKIN_DEPENDS
-    roscpp
-    autoware_perception_msgs
-    geometry_msgs
-    sensor_msgs
-    visualization_msgs
-)
-
-include_directories(include ${catkin_INCLUDE_DIRS})
+### Compile options
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic -Wno-unused-parameter -Werror)
```

Comment:
I think we all agreed that `-Werror` would not be included in this compile flags due to how strict it is

Reply:
Removed in 9e3e4ad, though the code seems to pass with `-Werror` enabled.